perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -340,6 +340,8 @@ def RunNetperf(vm, benchmark_name, server_ip, num_streams):
     throughput_stats = sample.PercentileCalculator(throughputs, [50, 90, 99])
     throughput_stats['min'] = min(throughputs)
     throughput_stats['max'] = max(throughputs)
+    # Calculate aggregate throughput
+    throughput_stats['total'] = throughput_stats['average'] * num_streams
```

Comment:
Looking at PercentileCalculator it finds the average by summing up the throughputs and dividing by the number of them.  Can you put an assert or something that checks to make sure that `num_streams` is equal to the length of `throughputs` ?